Sources/Orbit/Components/SegmentedSwitch.swift

```diff
@@ -173,6 +176,10 @@ public struct SegmentedSwitch<Selection: Hashable, Content: View>: View {
         idealSize.horizontal == true
     }
     
+    private var isSelected: Bool {
+        (selection as AnyObject) is NSNull == false
```

Comment:
Could you explain this one?

Reply:
This obscure workaround came from [here](https://forums.swift.org/t/how-to-check-if-a-value-of-a-generic-type-is-nil/50631/10).

Reply:
Let's add a comment about it then 👍

Reply:
Updated